### PR TITLE
Add example showing custom tokenizer for string tracking

### PR DESCRIPTION
**Pull request by lalmei**
## Description

Demo of custom tokenizer calls, and push along minor bug fixes

### General Checklist

* [x] Tests added for this feature/bug
      if it was a bug, test must cover it.
* [x] Conform by the style guides, by using formatter
* [x] Documentation updated
* [x] (optional) Please add a label to your PR

### CODE REVIEW

File: src/whylogs/app/logger.py [hunk 1]
```diff
@@ -299,6 +299,8 @@ def log(
         features: Optional[Dict[str, any]] = None,
         feature_name: str = None,
         value: any = None,
+        character_list: str = None,
```
Comment (andyndang) [inline src/whylogs/app/logger.py:302]: should the character list be tied to the profile/logger? It seems wrong to have this change across different `.log` call on the same logger?

Comment (lalmei) [reply src/whylogs/app/logger.py:302]: yeah it needs to stay the same otherwise there is an warning in string tracker. I can update to raise an error.

Comment (lalmei) [reply src/whylogs/app/logger.py:302]: This logic is already part of the original string tracker PR.

**Review by bernease — APPROVED** [review]
As discussed in person, this API stays but we'll think more carefully about it in the API redesign in the future.

**Comment by lalmei** [comment]
There was no change in the api here, it was already approved, api changes are in the roadmap already

**Pull request merged**